bundle/edu.gemini.itc/src/main/scala/edu/gemini/itc/service/ItcServiceImpl.scala

```diff
@@ -21,7 +22,9 @@ class ItcServiceImpl extends ItcService {
     }
   } catch {
     // TODO: for now in most cases where a validation problem should be reported to the user the ITC code throws an exception instead
-    case e: Throwable => ItcResult.forException(e)
+    case e: Throwable =>
+      e.printStackTrace()
```

Comment:
Me being lazy, sorry. But this will go away soon (once the error messages are displayed in a more useful way in the client).